examples/impute/plot_missing_values.py

```diff
@@ -5,42 +5,112 @@
 
 Missing values can be replaced by the mean, the median or the most frequent
 value using the basic :class:`sklearn.impute.SimpleImputer`.
-The median is a more robust estimator for data with high magnitude variables
-which could dominate results (otherwise known as a 'long tail').
 
-With ``KNNImputer``, missing values can be imputed using the weighted
-or unweighted mean of the desired number of nearest neighbors.
+In this example we will investigate different imputation techniques:
 
-Another option is the :class:`sklearn.impute.IterativeImputer`. This uses
-round-robin linear regression, treating every variable as an output in
-turn. The version implemented assumes Gaussian (output) variables. If your
-features are obviously non-Normal, consider transforming them to look more
-Normal so as to potentially improve performance.
+- imputation by the constant 0 value
+- imputation by the mean value of each feature combined with a missing-ness
+indicator auxiliary variable
+- k nearest neighbor imputation
+- iterative imputation
```

Comment:
you have a rendering pb see https://117-171681175-gh.circle-artifacts.com/0/doc/auto_examples/impute/plot_missing_values.html#sphx-glr-auto-examples-impute-plot-missing-values-py